packages/react-components/lib/reports/log-management/log-table.tsx

```diff
@@ -5,7 +5,12 @@ import { makeStyles, Typography } from '@material-ui/core';
 import moment from 'moment';
 import { materialTableIcons } from '../../material-table-icons';
 
-export type LogRowsType = { level: string; message: string; created: string }[];
+export type LogRowsType = {
+  level: string;
+  message: string;
+  created: string;
+  container_name: string;
```

Comment:
should this be camelCase?

Reply:
This is data coming from the backend so maybe it is a good idea to keep it as it is to differentiate from the FE pure data.